test/Java8andUp/src_90/org/openj9/test/java/lang/Test_System.java

```diff
@@ -162,25 +158,13 @@ public void test_getProperties() {
 		for (int i = 0; i < props.length; i++) {
 			AssertJUnit.assertTrue(props[i], System.getProperty(props[i]) != null);
 		}
-
```

Comment:
This test should stay. Its not testing java.version, but using java.version to determine when running Java 8, in order to check java.ext.dirs, which is still valid.

Reply:
Oh I see, nm, these tests are only for Java 9 and above.